CMSgov/mint-app
# [EASI-3919] - Notification settings

# EASI-3919


## Changes and Description

- Added route for notifications setting
- Added query and mutation for fetching and saving
- Added form components
- Error state
- LD Flag around route
- Unit tests

`taggedInDiscussionReply` is not currently being rendered in the UI, but it's data should always match `taggedInDiscussion` for now.

## How to test this change

- Navigate to `/notifications/settings`
- Verify all in-app notifications are always disabled
- Verify form persists
- Verify that `taggedInDiscussionReply` data always mirrors `taggedInDiscussion` when fetching/updating

## PR Author Review Checklist



## PR Reviewer Guidelines
- It's best to pull the branch locally and test it, rather than just looking at the code online!
- Check that all code is adequately covered by tests - if it isn't feel free to suggest the addition of tests.
- Always make comments, even if it's minor, or if you don't understand why something was done.
